google/cloud/spanner/samples/postgresql_samples.cc

```diff
@@ -165,10 +166,10 @@ void DmlGettingStartedUpdate(google::cloud::spanner::Client client) {
         auto budget1 = get_budget(txn, 1, 1);
         if (!budget1) return std::move(budget1).status();
         if (*budget1 < transfer_amount) {
-          return google::cloud::Status(
-              google::cloud::StatusCode::kUnknown,
+          return google::cloud::internal::UnknownError(
```

Comment:
we shouldnt use internal helpers in public samples

Reply:
Done